Yatima/Compiler/Utils.lean

```diff
@@ -1,4 +1,6 @@
 import Lean
+import Std
```

Comment:
I wrote this inport line but I should have imported a more specific file. Importing the entire Std is not efficient